CorsixTH/Src/th_gfx_font.h

```diff
@@ -87,17 +91,21 @@ class font
         @param iY The Y coordinate of the top-left corner of the bounding
             rectangle for the drawn text.
     */
-    virtual void draw_text(render_target* pCanvas, const char* sMessage,
-                           size_t iMessageLength, int iX, int iY) const = 0;
+    virtual void draw_text(
+            render_target* pCanvas,
+            const char* sMessage,
+            size_t iMessageLength,
+            int iX,
+            int iY) const = 0;
 
     //! Draw a single line of text, splitting it at word boundaries
     /*!
         This function still only draws a single line of text (i.e. any line
-        breaks like `\r` and `\n` in sMessage are ignored), but inserts line breaks
-        between words so that no single line is wider than iWidth pixels.
+        breaks like `\r` and `\n` in sMessage are ignored), but inserts line
+       breaks between words so that no single line is wider than iWidth pixels.
```

Comment:
Why is the indenting different?

Reply:
There's a few of these: seems that clang-format doesn't understand this style of comment.